maps/aurora/code/aurora_shuttles.dm

```diff
@@ -155,4 +155,16 @@
 	shuttle.area_offsite = locate(/area/shuttle/legion/centcom)
 	shuttle.area_station = locate(/area/shuttle/legion/station)
 	shuttles["Tau Ceti Foreign Legion"] = shuttle
+	START_PROCESSING(shuttle_controller, shuttle)
+
+	//Away Site shuttle.
+
+	shuttle = new()
+	shuttle.location = 0
+	shuttle.warmup_time = 10
+	shuttle.area_station = locate(/area/shuttle/research/station)
+	shuttle.area_offsite = locate(/area/shuttle/research/outpost)
```

Comment:
That should be changed to `/area/shuttle/research/away` so we dont have to modify `/outpost` (as mentioned above)